feconf.py

```diff
@@ -520,23 +520,23 @@ def get_empty_ratings():
 DEMO_EXPLORATIONS = {
     u'0': 'welcome.yaml',
     u'1': 'multiples.yaml',
-    u'2': 'binary_search',
+    u'2': 'binary_search.yaml',
```

Comment:
Why this change? I thought exploration files with assets were referred to without the yaml suffix.

Reply:
Oh! Thanks for the info, this wasn't known to me and I found admin page was looking bit weird so did it. I'll revert this changes. Thanks!

Reply:
Done!